src/ui/zcl_abapgit_gui_page_stage.clas.abap

```diff
@@ -252,6 +254,12 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_STAGE IMPLEMENTATION.
           ev_path     = ls_file-path
           ev_filename = ls_file-filename ).
 
+      READ TABLE ms_files-status ASSIGNING <ls_status>
+        WITH KEY
+          path     = ls_file-path
+          filename = ls_file-filename.
```

Comment:
Suggest using `WITH TABLE KEY` to optimize key access as it's a hashed table with keys path and filename. Same for others usages below.

Reply:
adjusted, though seems to be unnecessary  (see comments below)

Reply:
Interesting! I thought that omitting `TABLE` always leads to a linear search for hash tables. Obviously I was wrong. Thanks.
https://help.sap.com/doc/abapdocu_752_index_htm/7.52/en-US/abapread_table_free.htm
![image](https://user-images.githubusercontent.com/17437789/67630767-0c2f1480-f88d-11e9-934d-86b4bde86451.png)